Nos/Views/Discover/FeaturedAuthor.swift

```diff
@@ -229,3 +205,343 @@ extension FeaturedAuthor {
+        ),
+        FeaturedAuthor(
+            name: "Bill Bennett",
+            npub: "npub1gpcxn339ap3r9tju2uy6d5k8rmud7f8kr579w7zv54qyevgz9xsqdtr4us",
+            categories: [.news, .newzealand]
+        ),
+        FeaturedAuthor(
+            name: "The Spinoff",
+            npub: "npub1qngl40pxy06k3hrqp4ltkn4p5yacen7u93du58v4tamf73tzaqhs6dfy8g",
+            categories: [.news, .newzealand]
+        ),
+        FeaturedAuthor(
+            name: "Dan Selvin",
+            npub: "npub16jjukmhnvflj92dv2jr8z6udfhzyyuyf3mckmf6afws9w4xdhhzs745kmv",
+            categories: [.news, .newzealand]
+        ),
+        FeaturedAuthor(
+            name: "Emma Cook",
+            npub: "npub18pl5kgxl47e5pssxl9q38m2v7ez477gw3nz220jz2vslnjreqq8s2yxl9m",
+            categories: [.art, .newzealand]
+        ),
+        FeaturedAuthor(
+            name: "Jack Yan",
+            npub: "npub179lhhjz30t3n4utkkn26qzgmpx44v9s5qdtwjs5686px8ulw7dfszxmfgp",
+            categories: [.news, .newzealand]
+        ),
+        FeaturedAuthor(
+            name: "Lucire",
+            npub: "npub1l4s4mtt96znwu3plfeyupk37y63xfapw5e7kjn7uavuw02lvav5q3ngc3s",
+            categories: [.news, .newzealand]
+        ),
+        FeaturedAuthor(
+            name: "Ian Griffin",
+            npub: "npub19p6edn6lvz9dmegm7s8vyz9w5w0ejzgztke8rcswxgmqr35h8feszt36jc",
+            categories: [.scifi, .newzealand]
+        ),
+        FeaturedAuthor(
+            name: "JSM",
+            npub: "npub1uqeexjx2djkfwzxdnrnrrch5h2k4xn0uapcgsxm94ftaxrlhy5lqywjckg",
+            categories: [.tech]
+        ),
+        FeaturedAuthor(
+            name: "Carla",
+            npub: "npub1hu3hdctm5nkzd8gslnyedfr5ddz3z547jqcl5j88g4fame2jd08qh6h8nh",
+            categories: [.random]
+        ),
+        FeaturedAuthor(
+            name: "WaveLake",
+            npub: "npub1yfg0d955c2jrj2080ew7pa4xrtj7x7s7umt28wh0zurwmxgpyj9shwv6vg",
+            categories: [.music]
+        ),
+        FeaturedAuthor(
+            name: "Angela",
+            npub: "npub19vvkfwy9mcluhvehw7r56p4stsj5lmx4v9g3vgkwsm3arpgef8aqsrt562",
+            categories: [.random]
+        ),
+        FeaturedAuthor(
+            name: "J",
+            npub: "npub1fv9u4drq4hdrr7k45vn0krqy7mkgy8ajf059m0wq8szvcrsjlsrs8tdz3p",
+            categories: [.food, .random]
+        ),
+        FeaturedAuthor(
+            name: "Connie",
+            npub: "npub1468024mrwz6uhywjvt8s6vw4e604egnv8tfd2m2utrmqyd3nu3fsutvgjn",
+            categories: [.random]
+        ),
+        FeaturedAuthor(
+            name: "Ignacio",
+            npub: "npub1wptpq88jyjv6nwwltqx0exhxsg8zt0dcm4uwvtcxn80hxk2r66hswdm8rf",
+            categories: [.lifestyle, .espanol]
+        ),
+        FeaturedAuthor(
+            name: "Nacho",
+            npub: "npub1daajnadf0f0s7uz3yftur8434rtz2s949gkdpx7uyeapm9rlt0qq9q8w5z",
+            categories: [.lifestyle]
+        )
+    ]
+}
+
+extension FeaturedAuthor {
+    static let selectNew = [
```

Comment:
I did this because the authors featured in the `new` category are not all the newly added authors.

Reply:
Maybe we just call this `new`, instead of `selectNew`? That way it matches the category name in the UI.

Reply:
I was running away from using the SwiftLint ignore 3 words name 😄

Reply:
Ah, yes, that makes sense. Maybe `newAuthors`?

Reply:
But they are not the only new authors. They are selected from cohort5 which are all new.

Reply:
what about `selectedNewAuthors`?